dd-sdk-android/src/main/kotlin/com/datadog/android/sessionreplay/internal/SessionReplayFeature.kt

```diff
@@ -32,7 +30,7 @@ internal class SessionReplayFeature(
                 SessionReplayRumContextProvider(sdkCore),
                 configuration.privacy,
                 recordWriter,
-                SessionReplayTimeProvider(coreFeature.contextProvider),
+                SessionReplayTimeProvider(sdkCore),
```

Comment:
I did use the `sdkCore` before here but I had problems as it was using the `NoOpSdkCore`. For some reason when using the lazy instantiation it was keeping the initial instance which was the `NoOpSdkCore`. Can you please verify that is not the case now.

Reply:
It is all good, `SessionReplayFeature` is created from `DatadogCore` only:

![image](https://user-images.githubusercontent.com/4046447/203512197-f0b3194b-27fc-4fe7-bb6a-a4d784363928.png)

No different from the `SessionReplayRumContextProvider` anyway which also takes `sdkCore`.